### DISCUSSION TITLE
Custom deployment

### DISCUSSION THREAD

**Discussion by tuan231195** [vercel/vercel#4061] [General]
I want to deploy a serverless nextjs application written using pnpm workspace to now. The application has a custom installation and custom build steps. Is it possible to tell the now CLI to deploy an already built .next folder?

**Comment by flybayer** [comment]
I have this exact same question.

How can I tell now that my project is a next.js build with the following?
- Next.js is not a dependency of the project
- My custom build output is a serverless Next build in `.next`

**Reply by cyrus-za** [reply]
I just found this discussion thread while searching for a fix to a pnpm issue on vercel, and then saw your post @flybayer and it warmed my heart knowing this app/thing you were referencing above are the early days of blitz.js <3

**Reply by flybayer** [reply]
@cyrus-za :D
